### DISCUSSION TITLE
README has broken link to "Scanning Coverage" page

### DISCUSSION THREAD

**Discussion by ghost** [aquasecurity/trivy#5109] [Documentation]
### Description

The link in the top-level README.md to "Scanning Coverage" (target: https://aquasecurity.github.io/trivy/latest/getting-started/coverage/ ) is broken.

It looks like that page no longer exists for 0.45, hence why there's no redirect for the "latest" URI.

### Link

https://github.com/aquasecurity/trivy/blob/main/README.md

### Suggestions

Perhaps the link should be updated to point to https://aquasecurity.github.io/trivy/latest/docs/coverage/ as that works for the latest versions.

**Comment by DmitriyLewen** [comment]
Hello @Rob-HCL 
Thanks for your report!

Created #5203 to fix link.

Regards, Dmitriy

**Reply by knqyf263** [reply]
Merged
